challenge-manager/edge-tracker/challenge_confirmation.go

```diff
@@ -167,7 +168,7 @@ func (cc *challengeConfirmer) beginConfirmationJob(
 	}
 
 	onchainInheritedTimer, err := retry.UntilSucceeds(ctx, func() (protocol.InheritedTimer, error) {
-		timer, innerErr := royalRootEdge.SafeHeadInheritedTimer(ctx)
+		timer, innerErr := royalRootEdge.LatestInheritedTimer(ctx)
```

Comment:
The internal call to the solidity will use the configured block number, so it will still use `safe` underneath the hood if the user desires

Reply:
I feel like this would be a good candidate for some in-code documentation (a.k.a. a comment.)